database.js

```diff
@@ -36,11 +36,10 @@ Database.prototype = {
     const addressId = signature.getAddressId(address)
 
     const options = {
-      json: true,
       method: 'GET',
       url: `${this.resolveEndpoint('address')}/${address}`,
       cacheKey: ['address', addressId].join('_'),
-      qs: params
+      params: params
```

Comment:
We could use shorthand properties here ( so you don't have to do `{ params: params }` but just `{ params }`)